neqo-http3/src/features/extended_connect/webtransport_session.rs

```diff
@@ -486,6 +486,10 @@ impl SendStream for Rc<RefCell<WebTransportSession>> {
         self.borrow_mut().has_data_to_send()
     }
 
+    fn set_sendorder(&mut self, _conn: &mut Connection, _sendorder: Option<SendOrder>) {
+	// Not relevant on session
+    }
```

Comment:
I think that we should let this pass through.  We will simply not call it in gecko (who knows, maybe this makes sense at some point).

Reply:
Pass through to where?

Reply:
I am suggesting that you change the send order, rather than do nothing

Reply:
Sessions don't have a sendorder to set, so I'm still confused by your suggestion.   Unless you mean "set the sendorder of all streams to N", which would be unhelpful and there's no reason for it.

Reply:
Maybe add a debug assertion here?
I assume this function should not be called at all.